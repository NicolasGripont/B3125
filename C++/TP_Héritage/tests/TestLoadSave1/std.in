S seg1 1 1 7 5
R rect1 2 1 11 5
OR or1 seg1 rect1
PC pc1 0 0 4 8 8 -1
OI oi1 or1 pc1
SAVE figures.txt
LOAD figures1.txt
CLEAR
LOAD figures1.txt
LIST
CLEAR
SAVE vide.txt
LIST
LOAD figures1.txt
LIST
CLEAR
LIST
LOAD figures2SameNames.txt
LOAD figuresWithError.txt
EXIT