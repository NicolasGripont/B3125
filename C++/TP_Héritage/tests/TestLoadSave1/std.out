OK
OK
OK
OK
OK
OK
ERR
OK
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OI oi1 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
OK
OK
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OI oi1 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
OK
ERR
ERR